.github/workflows/deploy.yml

```diff
@@ -0,0 +1,71 @@
+name: Build and deploy
+
+on:
+  push:
+    branches: [ "release-*"]
+
+env:
+  AWS_REGION: eu-west-2
+  ECS_SERVICE: ak-python-ecs-service
+  ECS_CLUSTER: ak-python-ecs-cluster
+  ECS_TASK_DEFINITION: task-definition.json
+  CONTAINER_NAME: ak-python-container
+
+
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    permissions:
+          contents: read
+          packages: write
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
```

Comment:
is v1 the latest version

Reply:
yes still on v1.x.x and read that using v1 tracks the latest v1 release